person.rb

```diff
@@ -1,39 +1,33 @@
-require './nameable'
-require './book'
-require './rental'
+require_relative 'nameable'
+require_relative 'rental'
 
 class Person < Nameable
   attr_accessor :name, :age
   attr_reader :id, :rentals
 
-  def initialize(age, name = 'unknown', parent_permision: true)
+  def initialize(age, name = 'Unknown', parent_permission: true)
     super()
     @id = Random.rand(1..1000)
```

Comment:
- [ ] An error occurs when I attempt to create a person. As a result I am unable to check other functionalities of your app such as creating a rental or listing all rentals for a specific person id. Kindly see error below 👇  and fix it 🙏 
![image](https://user-images.githubusercontent.com/46839548/175431842-fe986913-5507-42f6-bbf4-494b14c176ef.png)